en/features/graphql.md

```diff
@@ -1,22 +1,22 @@
 # API Documentation
 
-* [Characteristics](#characteristics)
-* [GraphQL](#graphql)
-* [Making API requests](#making-api-requests)
-  * [Supported clients](#supported-clients)
-    * [GraphiQL](#graphiql)
-    * [Postman](#postman)
-    * [HTTP libraries](#http-libraries)
-* [Available information](#available-information)
-* [Examples of queries](#examples-of-queries)
-  * [Request a single record from a collection](#request-a-single-record-from-a-collection)
-  * [Request a complete collection](#request-a-complete-collection)
-    * [Pagination](#pagination)
-  * [Accessing several resources in a single request](#accessing-several-resources-in-a-single-request)
-* [Security limitations](#security-limitations)
-  * [Example of too deep query](#example-of-too-deep-query)
-  * [Example of too complex query](#example-of-too-complex-query)
-* [Code examples](#code-examples)
+- [Characteristics](#characteristics)
+- [GraphQL](#graphql)
+- [Making API requests](#making-api-requests)
+  - [Supported clients](#supported-clients)
+    - [GraphiQL](#graphiql)
+    - [Postman](#postman)
+    - [HTTP libraries](#http-libraries)
+- [Available information:](#available-information)
+- [Examples of queries](#examples-of-queries)
+  - [Request a single record from a collection](#request-a-single-record-from-a-collection)
+  - [Request a complete collection](#request-a-complete-collection)
+    - [Pagination](#pagination)
+  - [Accessing several resources in a single request](#accessing-several-resources-in-a-single-request)
+- [Security limitations](#security-limitations)
+  - [Example of too deep query](#example-of-too-deep-query)
+  - [Example of too complex query](#example-of-too-complex-query)
+- [Code examples](#code-examples)
```

Comment:
I think we use `*` everywhere else to display a list of links :thinking:. Why the change here?

Reply:
I didn't change it actually: VS Code convert it in that way when I save some changes in files containing tables of contents. Very strange behavior!